src/routes/business/businesses.ts

```diff
@@ -261,3 +262,27 @@ export const updateBusiness = async (req, res) => {
 
   return res.status(200).send(business);
 };
+
+export const getBusinessAccount = async (req: any, res: any) => {
+  const { id } = req.params;
+  const {
+    business: { account },
+  } = req;
+
+  if (!account) {
+    res.status(404).send({
+      errors: [
+        {
+          id: generateID(),
+          status: 404,
+          code: "model_not_found",
+          title: "Model Not Found",
+          detail: `Couldn't find 'Account' for id '${id}'.`,
+        },
+      ],
+    });
+    return;
+  }
+
+  res.status(201).send(account);
```

Comment:
`201` is for a successful creation of an entity 😄 

```suggestion
  res.status(200).send(account);
```